### DISCUSSION TITLE
How to annotate a column as the source field for multiple vector columns

### DISCUSSION THREAD

**Discussion by humpydonkey** [lancedb/lancedb#1170] [Q&A]
Hi team,

I just have a quick question:  how to annotate a LanceModel field/column as the source field for multiple vector columns.

For example, in the below schema, the Image table stores the raw image as well as two embeddings for the image column.
I couldn't find any docs from the official documentation.

```
class Image(LanceModel):
    image_uri: str
    vec_clip: Vector(clip.ndims()) = clip.VectorField()
    vec_dino: Vector(dino.ndims()) = dino.VectorField()
```

Thanks!

**Comment by AyushExel** [comment]
This usage should be possible. Are you seeing any errors?
